.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,35 @@
-## Type of change
+## 🎟️ Tracking
 
-<!-- (mark with an `X`) -->
+<!-- Paste the link to the Jira or GitHub issue or otherwise describe / point to where this change is coming from. -->
 
-```
-- [ ] Bug fix
-- [ ] New feature development
-- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
-- [ ] Build/deploy pipeline (DevOps)
-- [ ] Other
-```
+## 📔 Objective
 
-## Objective
+<!-- Describe what the purpose of this PR is, for example what bug you're fixing or new feature you're adding. -->
 
-<!--Describe what the purpose of this PR is. For example: what bug you're fixing or what new feature you're adding-->
+## 📸 Screenshots
```

Comment:
The SDK itself probably won't but it can sometimes be helpful for the secrets manager CLI (which is part of this repo) to have screenshots when the terminal commands get updated.

For example:
https://github.com/bitwarden/sdk/pull/320, https://github.com/bitwarden/sdk/pull/103, https://github.com/bitwarden/sdk/pull/175